itest/litd_mode_integrated_test.go

```diff
@@ -0,0 +1,25 @@
+package itest
+
+import (
+	"context"
+	"time"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/stretchr/testify/require"
+)
+
+// testModeIntegrated makes sure that in integrated mode all daemons work
+// correctly.
+func testModeIntegrated(net *NetworkHarness, t *harnessTest) {
+	ctx := context.Background()
+
+	time.Sleep(2 * time.Second)
```

Comment:
Is it possible to move this synchronization up to where the tests themselves are actually run?